### ISSUE TITLE
Plugins in Discourse not working even though they appear to be installed correctly

### ISSUE THREAD

**Issue by Dehumanizer77** [NixOS/nixpkgs#127795]
**Describe the bug**
I have tried to install two plugins, namely discourse-solved and discourse-canned-replies, none of them work, but they appear in the plugin list as installed and enabled.

**To Reproduce**
1. install plugins by setting services.discourse.plugins to:
[
        (pkgs.fetchFromGitHub {
          owner = "discourse";
          repo = "discourse-solved";
          rev = "ca0dbd6d4c99958b88a1c9ec0661728ddac468b1";
          sha256 = "02blzbiwkcqb7nfw0llj9m081w6iixldshmrf3s9z9315zlnxr31";
        })
        (pkgs.fetchFromGitHub {
          owner = "discourse";
          repo = "discourse-canned-replies";
          rev = "7ee748f18a276aca42185e2079c1d4cadeecdaf8";
          sha256 = "0j10kxfr6v2rdd58smg2i7iac46z74qnnjk8b91jd1svazhis1ph";
        })
   ];
2. after rebuilding, check the admin/plugins section of discourse and you see the plugins are there and enabled (see the screenshot bellow)
3. check to open a new reply and you should have the "canned replies" in the gear menu (provided by discourse-canned-replies plugin), but it doesn't
4. when you click "show more" bellow any reply in the topic, you should see the "solution" button (provided by discourse-solved plugin), but it's not there

**Expected behavior**
explained above

**Screenshots**
admin/plugins screen:
![1](https://user-images.githubusercontent.com/1876451/122925999-b19ed200-d367-11eb-836a-ecb838e2fe3b.png)

what it shows in the reply after installation:
![2](https://user-images.githubusercontent.com/1876451/122926005-b2376880-d367-11eb-91ba-9455bfb07d05.png)

what it should show in the reply after installation:
![3](https://user-images.githubusercontent.com/1876451/122926008-b2376880-d367-11eb-9824-1509c8c02e5a.png)

what it shows bellow a reply:
![4](https://user-images.githubusercontent.com/1876451/122926013-b2376880-d367-11eb-9f57-7dd561ebe45d.png)

what it should show bellow a reply after installation:
![5](https://user-images.githubusercontent.com/1876451/122926070-c8ddbf80-d367-11eb-8e56-34f09e938ce6.png)

**Additional context**
N/A

**Notify maintainers**

@talyz 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.43, NixOS, 21.11pre-git (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.12`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by talyz** [comment]
Thanks for reporting this! I've submitted a fix in #127931, so please try it out and make sure all your plugins work as expected.

**Comment by Dehumanizer77** [comment]
Thanks! I confirm the fix works for me and both plugins I have installed are working fine.
